modules/transactions/src/testFixtures/java/org/apache/ignite/internal/tx/storage/state/AbstractTxStateStorageTest.java

```diff
@@ -135,6 +158,45 @@ private HybridTimestamp generateTimestamp(UUID uuid) {
         return hybridTimestamp(time);
     }
 
+    @Test
+    public void testRemoveAll() {
```

Comment:
Do we check that lastAppliedIndex and lastAppliedTerm were properly updated in case of removeAll?